azure-pipelines.yml

```diff
@@ -600,6 +600,39 @@ jobs:
         inlineScript: |
           Get-ChildItem .vault-config/*.yaml |% { dotnet secret-manager synchronize $_}
 
+  # Ubuntu 1804 x64 ILLink Benchmarks
+  - template: /eng/performance/benchmark_jobs.yml
+    parameters:
+      osName: ubuntu
+      osVersion: 1804
+      kind: illink
+      architecture: x64
+      pool:
+        vmImage: ubuntu-latest
+      machinePool: Tiger
+      queue: Ubuntu.1804.Amd64.Tiger.Perf # using a dedicated private Helix queue (perftigers)
+      container: ubuntu_x64_build_container
+      csproj: src/benchmarks/real-world/ILLink/ILLinkBenchmarks.csproj
+      runCategories: 'illink'
+      channels: # for private jobs we want to benchmark .NET Core 3.1 and 5.0 only
+        - main
+
+  # Windows x64 ILLinkBenchmarks
+  - template: /eng/performance/benchmark_jobs.yml
+    parameters:
+      osName: windows
+      osVersion: 22H1
+      kind: illink
+      architecture: x64
+      pool:
+        vmImage: windows-2019
+      machinePool: Tiger
+      queue: Ubuntu.1804.Amd64.Tiger.Perf # using a dedicated private Helix queue (perftigers)
```

Comment:
This queue name needs to be updated to the actual Windows queue name, `Windows.10.Amd64.19H1.Tiger.Perf`